src/desktop/apps/auction_support/routes.coffee

```diff
@@ -29,12 +31,16 @@ registerAndRedirect = (sale, req, res, next) ->
                 }
               )
               res.redirect sale.registrationSuccessUrl()
+      else
+        # FIXME: Not sure how to handle this case
+        console.log('Error: No credit card...')
+        res.redirect sale.redirectUrl(sale.get('href'))
```

Comment:
Redirecting back to the sale if no card as a quick-fix.

Reply:
Very strange to me that we hadn't already accounted for this case elsewhere, or that it was even possible to access in this condition. 🤷‍♀️